File: co.edu.uniandes.fixml.toCSharp/test/test3.xml/OrderQuantity.cs

using System.Collections.Generic;
// Class for tag OrderQuantity
class OrderQuantity {


	List<OrderQty> _OrderQtys = new List<OrderQty>();
	
	// default constructor
	
	public OrderQuantity ( ) {   
		;
		
		OrderQty OrderQuantity_OrderQty= new OrderQty();
		this.AddOrderQty( OrderQuantity_OrderQty );
		
		
	}
	
	public OrderQuantity (  ) {
	
	}
	
	public void AddOrderQty( OrderQty child ) {
		this._OrderQtys.Add( child );
	}
	
	
} 

File: co.edu.uniandes.fixml/test/co/edu/uniandes/fixml/src-gen/test4/Java/OrderQuantity.java
package object;

import java.util.List;
import java.util.ArrayList;

// Class for tag OrderQuantity
public class OrderQuantity {


  List<OrderQty> OrderQtys = new ArrayList<OrderQty>();



  public OrderQuantity ( ) {   
  
;
  
OrderQty OrderQuantity_OrderQty_0= new OrderQty();
this.addOrderQty( OrderQuantity_OrderQty_0 );
  		
  }
      
  public void addOrderQty( OrderQty child ) {
    this.OrderQtys.add( child );
  }

} 